Lib/test/test_functools.py

```diff
@@ -2310,7 +2310,7 @@ def test_invalid_positional_argument(self):
         def f(*args):
             pass
         msg = 'f requires at least 1 positional argument'
-        with self.assertRaisesRegexp(TypeError, msg):
+        with self.assertRaises(TypeError, msg=msg):
```

Comment:
This is not correct use of *msg*. The correct code is:
```python
with self.assertRaisesRegex(TypeError, msg):
```
The *msg* argument of `assertRaises()` is used for reporting a failure. But it should be used for testing the error message.

Reply:
If the code looks wrong, you should reopen https://bugs.python.org/issue33967 and comment there, otherwise your comment may be lost.

Reply:
I'm sorry.  It was my mistake.